src/app/enums/acl-type.enum.ts

```diff
@@ -24,3 +24,49 @@ export enum AclMode {
   Passthrough = 'PASSTHROUGH',
   Discard = 'DISCARD',
 }
+
+export enum DefaultAclType {
+  Nfs4Open = 'NFS4_OPEN',
+  Nfs4Restricted = 'NFS4_RESTRICTED',
+  Nfs4Home = 'NFS4_HOME',
+  Nfs4DomainHome = 'NFS4_DOMAIN_HOME',
+  PosixOpen = 'POSIX_OPEN',
+  PosixRestricted = 'POSIX_RESTRICTED',
+  Open = 'OPEN',
+  Restricted = 'RESTRICTED',
+  Home = 'HOME',
+}
+
+export enum DAclNfs4Tag {
+  Owner = 'owner@',
+  Group = 'group@',
+  Everyone = 'everyone@',
+  User = 'USER',
+  UserGroup = 'GROUP',
+}
+
+export enum DAclType {
+  Allow = 'ALLOW',
+  Deny = 'DENY',
+}
+
+export enum DAclPermissionsBasic {
+  FullControl = 'FULL_CONTROL',
+  Modify = 'MODIFY',
+  Read = 'READ',
+  Traverse = 'TRAVERSE',
+}
+
+export enum DAclFlagsBasic {
+  Inherit = 'INHERIT',
+  NoInherit = 'NOINHERIT',
+}
+
+export enum DAclPosix1eTag {
```

Comment:
Looks same with AclItemTag https://github.com/truenas/webui/blob/e29b9adbd134e3ca203b486d7c3838ddcdade08d/src/app/enums/acl-type.enum.ts#L7
is that intended to have two different?